### ISSUE TITLE
Support for Regex in Paths 

### ISSUE THREAD

**Issue by falitjain** [nginxinc/kubernetes-ingress#782]
Hi, 
Does the ingress controller has now support for regex in paths?
https://github.com/nginxinc/kubernetes-ingress/issues/20 is the same request, but in OPEN state, was it developed and and if yes: 

- Does it require any annotation
- Which version was it introduced?

**Comment by pleshakov** [comment]
hi @falitjain 

We don't support regex in Ingress resources. 

However, we recently added support for regex to our [VirtualServer/VirtualServerRoute](https://github.com/nginxinc/kubernetes-ingress/blob/master/docs/virtualserver-and-virtualserverroute.md) resources.  Here is an example:
```yaml
  - path: ~ ^/decaf/.*\\.jpg$
    action:
      pass: coffee
```

This feature will be available in 1.6.0 release, which is planned for the middle of December. Currently, it is already available in the edge version -- https://github.com/nginxinc/kubernetes-ingress/#nginx-ingress-controller-releases

**Comment by Dean-Coakley** [comment]
Added in: https://github.com/nginxinc/kubernetes-ingress/pull/766
